source/MaterialXContrib/MaterialXNode/MaterialXData.cpp

```diff
@@ -68,12 +70,12 @@ void MaterialXData::createDocument(const std::string& materialXDocumentPath)
 
     // Load libraries
     static const mx::StringVec libraries = { "stdlib", "pbrlib", "bxdf", "stdlib/genglsl", "pbrlib/genglsl" };
-    mx::loadLibraries(libraries, _librarySearchPath, _document);
+    MaterialXMaya::loadLibraries(libraries, _librarySearchPath, _document);
 
     // Read document contents from disk
     mx::XmlReadOptions readOptions;
     readOptions.skipDuplicateElements = true;
-    mx::readFromXmlFile(_document, materialXDocumentPath);
+    mx::readFromXmlFile(_document, materialXDocumentPath, mx::EMPTY_STRING, &readOptions);
```

Comment:
Duplicate element load throws exceptions. Was missing setting option to not do so.